src/artemis/system_tests/test_main_system.py

```diff
@@ -467,3 +463,9 @@ def test_warn_exception_during_plan_causes_warning_in_log(
     assert response_json["message"] == 'WarningException("D\'Oh")'
     assert response_json["exception_type"] == "WarningException"
     assert caplog.records[-1].levelname == "WARNING"
+
+
+def test_when_context_created_then_contains_expected_number_of_plans():
```

Comment:
Isn't this a bit needlessly brittle? Any time we add a plan we'll now need to adjust the expected number of plans here?

Some options to consider:
- Check that a few known plans are in the `plan_functions` list
- Mock out some of the import machinery so that we can mock the expected plans in the test, and therefore they won't rely on the actual plans

Reply:
Good spot, thanks!